Utils/Dataflow/pyDKB/common/json_utils.py

```diff
@@ -1,14 +1,39 @@
 """
-Utils to work with JSON (dict) objects.
+Utils to work with JSON objects.
+
+In context of Python, JSON [#]_ objects may be considered as structures
+consisting of six types of elements:
+
+- dictionaries,
+- lists,
+- strings,
+- numbers,
+- True/False,
+- Null.
+
+DKB project uses JSON for storing various information and transferring it
+between stages. This module contains functions which simplify some aspects
+of retrieving data from JSON objects.
+
+.. [#] https://www.json.org/
+
 """
 
 
 def valueByKey(json_data, key):
     """ Return value by a chain (list) of nested keys.
 
-    Parameters:
-        DICT   json_data -- to search in
-        STRING key       -- dot-separated list of nested keys
+    It is common for JSON objects to contain many layers of dictionaries
+    nested in other dictionaries -- this function extracts the data from
+    such constructions according to given string or list with keys.
```

Comment:
The description does not explain how keys should be passed. If usage of list version can be guessed, string is not clear at all.
What about something like this?

```
...according to a list of keys. 
<...>
:param keys: sequence of nested keys (dot-separated string or list object)
```
?

Reply:
Makes sense, will do.